### PR TITLE
Add recommendation for audio devices

### PR DESCRIPTION
**Pull request by ann0see**
see https://github.com/jamulussoftware/jamuluswebsite/pull/169#issuecomment-742836574

### CODE REVIEW

**Comment by ignotus666** [comment]
Just a comment: you don't need the hyphen in 'DVD quality' and 'ASIO driver'.

**Comment by ann0see** [comment]
removed them

File: wiki/en/en-Getting-Started.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ Also - [Jamulus OS!](https://sourceforge.net/projects/jamulus-os/files/JamulusOS
 
 ## Recommended hardware
 
-See [the list of known good hardware](Sound-Devices) and [Hardware Setup](Hardware-Setup).
+In general, your device must support a sample rate of **48kHz** (DVD quality). On Windows it's recommended to choose a device with a native [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output) driver.
+
```
Comment (gegeweb) [inline wiki/en/en-Getting-Started.md:43]: I would add something like this here:
Your device should also allow you to listen to the computer's audio signal. (Jamulus server mix monitoring)

Comment (ann0see) [reply wiki/en/en-Getting-Started.md:43]: Added. Can still be discussed

**Review by gilgongo — APPROVED** [review]
Nice!

**Comment by ann0see** [comment]
Merge?

**Pull request merged**